src/SmartComponents/SystemAdvisor/SystemAdvisor.js

```diff
@@ -464,7 +465,15 @@ const BaseSystemAdvisor = () => {
 
       setKbaDetailsData(kbaDetailsFetch);
       setRows(
-        buildRows(reportsData, kbaDetailsFetch, filters, rows, searchValue)
+        buildRows(
+          reportsData,
+          kbaDetailsFetch,
+          filters,
+          rows,
+          searchValue,
+          false,
```

Comment:
Currently I dont see exactly what the point of the kbaLoading arg is, its never used in any buildRow calls. I have another PR in the making for an advisor refactor, and will look into this more in that PR.